tools/python/maps_generator/generator/settings.py

```diff
@@ -12,6 +12,9 @@
 from maps_generator.utils.md5 import md5_ext
 from maps_generator.utils.system import total_virtual_memory
 
+def red(text):
+    return '\033[91m'+ text+ '\033[0m'
```

Comment:
Not all terminals support ansi codes. Is there a way to check it (e.g. env var TERM=xterm-256color, or some built-in python method? Otherwise just printing normal text would be enough.

Reply:
didn't find any built-in python method, i think just printing plain text will be fine 👍️